src/staticobject.cpp

```diff
@@ -75,6 +75,7 @@ void StaticObjectList::serialize(std::ostream &os)
 }
 void StaticObjectList::deSerialize(std::istream &is)
 {
+	m_stored.clear();
```

Comment:
Active objects are created on the fly, they are not loaded directly and therefore deserializing does not modify them.